components/button/button-icon.js

```diff
@@ -51,9 +54,21 @@ class ButtonIcon extends ButtonMixin(VisibleOnAncestorMixin(RtlMixin(LitElement)
 				:host([hidden]) {
 					display: none;
 				}
+				:host([translucent]) {
```

Comment:
I refactored "translucent" button-icon styles while I was in here, as it could take advantage of the new CSS variables.